src/components/media-loader.js

```diff
@@ -28,7 +28,18 @@ AFRAME.registerComponent("media-loader", {
     src: { type: "string" },
     resize: { default: false },
     resolve: { default: false },
-    contentType: { default: null }
+    contentType: { default: null },
+    mediaOptions: {
+      default: {},
+      parse: value => {
+        if (typeof value === "object") {
```

Comment:
super minor but I would write:
```
parse: v => (typeof v === "object" ? JSON.parse(v) : v)
```

Reply:
In this case I think that's fine because it is short and readable. I generally avoid ternary operators when the cases include something other than a value (In this case, `JSON.parse`). They can be hard to read and minification will reduce the size of the function anyway.

Reply:
It's not about being less code, its about avoiding multiple codepaths and making as many things possible an expression.